src/components/shared/datePicker/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import DatePicker from 'react-native-date-picker';
+
+interface DatePickerInputProps {
+  date: Date;
+  open: boolean;
+  mode?: 'date' | 'time' | 'datetime' | undefined;
+  onConfirm: () => void;
+  onCancel: () => void;
```

Comment:
perhaps it might be a good idea to make this prop optional, what do you think?

Reply:
Same